### PR TITLE
Snapshot bump action

### PR DESCRIPTION
**Pull request by matJC**

### CODE REVIEW

**Comment by msandrofJC** [comment]
You might want to check with @matJC about this. He's looking at using some standard tools to do some of this. Otherwise, looks fine.

File: snapshot-bump/snapshot-bump.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  INPUT_PROPERTY=packageVersion
+  echo "using default property: packageVersion"
+fi
+
+if ! [[ `grep $INPUT_PROPERTY $INPUT_FILE` ]]; then
+  echo "property $INPUT_PROPERTY not found in file $INPUT_FILE. bailing"
+  help;
+fi
+
+VERSION_PROP=`grep packageVersion gradle.properties` # ex. packageVersion=1.7.8
```
Comment (pdstrnadJC) [inline snapshot-bump/snapshot-bump.sh:54]: Should this be:
```
VERSION_PROP=`grep $INPUT_PROPERTY $INPUT_FILE`
```

**Comment by pdstrnadJC** [comment]
Looks good overall. 

Curious about when to use a Docker-based action vs composite - when is one preferable over the other?

File: snapshot-bump/snapshot-bump.sh (hunk 1 above)
Comment (matJC) [reply snapshot-bump/snapshot-bump.sh]: Yes! Thanks
https://github.com/mindjolt/actions/pull/2/commits/2654b01670ee4dbfdbc25c763b4d332a3e395172

**Pull request merged**